### ISSUE TITLE
Failed to connect to binary FirefoxBinary

### ISSUE THREAD

**Issue by kamal2222ahmed** [nikulkarni/webdriver-accessibility#7]
mvn clean -B deploy
results in error:

simpleWebDriverAccessibilityTest(com.accessibility.examples.junit.webdriver.test.WebDriverJunitTest): Failed to connect to binary FirefoxBinary(/Applications/Firefox.app/Contents/MacOS/firefox-bin) on port 7055; process output follows: (..)
